### PR TITLE
Merge feature.color-4 into main

### PR DESCRIPTION
**Pull request by nex3**
See https://github.com/sass/dart-sass/issues/1805
See https://github.com/sass/sass/pull/3930
See https://github.com/sass/sass-spec/pull/2013
[skip sass-embedded]

### CODE REVIEW

File: lib/src/value/color/channel.dart [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  @internal
+  const ColorChannel(this.name,
+      {required this.isPolarAngle, this.associatedUnit});
+
+  /// Returns whether this channel is [analogous] to [other].
+  ///
+  /// [analogous]: https://www.w3.org/TR/css-color-4/#interpolation-missing
+  bool isAnalogous(ColorChannel other) => switch ((name, other.name)) {
+        ("red" || "x", "red" || "x") ||
+        ("green" || "y", "gren" || "y") ||
```
Comment (jathak) [inline lib/src/value/color/channel.dart:45]:
```suggestion
        ("green" || "y", "green" || "y") ||
```

File: test/dart_api/value/color_test.dart [hunk 2]
```diff
@@ -2,6 +2,7 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+@Skip("TODO(nweiz): Update these for the new Color API")
```
Comment (jathak) [inline test/dart_api/value/color_test.dart:5]: Should this have a tracking bug?

**Review by jathak — APPROVED** [review]

File: test/dart_api/value/color_test.dart (hunk 2 above)
Comment (nex3) [reply test/dart_api/value/color_test.dart]: Oops! I've just updated these in this PR, PTAL

**Review by jathak — APPROVED** [review]

**Comment by nex3** [comment]
I think the CI is going weird here because GitHub can't tell the difference between the PR actions and the normal branch actions, and the branch actions aren't handling the link with the language repo properly. I'm just going to merge and clean up any failures after the fact.

**Comment by stof** [comment]
Actually, GitHub can differentiate them. But the dropdown showing the list has a max-width and truncates the names of the statuses when they are too long to fit, and the info to distinguish the status being set by `(push)` or `(pull_request)` is at the end of the name.

**Pull request merged**